apps/meteor/ee/app/livechat-enterprise/lib/messageTypes.js

```diff
@@ -7,16 +7,17 @@ MessageTypes.registerType({
 	system: true,
 	message: 'New_chat_priority',
 	data(message) {
-		if (!message.priorityData) {
+		if (!message.slaData) {
 			return;
 		}
 		const {
 			definedBy: { username },
-			priority: { name = null } = {},
-		} = message.priorityData;
+			priority: { name: priorityName = null } = {},
```

Comment:
Does it make sense to leave both sla and priority here?

Reply:
we'll use priorities in the feature, this PR is mostly to open the name priorities.